picoCTF-web/api/apps/v1/users.py

```diff
@@ -99,3 +99,26 @@ def delete(self, user_id):
         return jsonify({
             'success': True
         })
+
+
+@ns.response(200, 'Success')
+@ns.response(401, 'Not logged in')
+@ns.response(403, 'Not authorized')
+@ns.response(404, 'User not found')
+@ns.route('/<string:user_id>/export')
+class User(Resource):
```

Comment:
let's call this `UserDataExport` similar to the logged-in one – otherwise LGTM!

Reply:
Oops, it's a copy & paste mistake that didn't raise any errors.